sycl/test/check_device_code/id_queries_fit_int.cpp

```diff
@@ -1,51 +1,56 @@
-// RUN: %clangxx -fsycl -fsycl-id-queries-fit-in-int -O0 -c -S -emit-llvm -o %t.ll %s
+// RUN: %clangxx -D__SYCL_DISABLE_PARALLEL_FOR_RANGE_ROUNDING__ -fsycl-device-only -fsycl-id-queries-fit-in-int -fno-sycl-early-optimizations -S -o %t.ll %s
 // RUN: FileCheck %s --input-file %t.ll
 
 #include <CL/sycl.hpp>
 
 using namespace sycl;
 
-// CHECK: define dso_local i32 @main() {{.*}} {
 int main() {
```

Comment:
Apply `SYCL_EXTERNAL` macro to the test function instead of using runtime queue and setting `__SYCL_DISABLE_PARALLEL_FOR_RANGE_ROUNDING__` to emit device code.

Reply:
Good catch!